src/index.ts

```diff
@@ -15,3 +19,18 @@ export function convertAxeToSarif(axeResults: Axe.AxeResults): SarifLog {
     const sarifConverter = defaultSarifConverter();
     return sarifConverter.convert(decoratedAxeResults, {});
 }
+
+export function sarifReporter(
+    rawResults: AxeRawResult[],
+    {},
+    callback: Function,
+) {
+    const environmentData: EnvironmentData = getEnvironmentDataFromEnvironment();
+    callback(() => {
```

Comment:
`callback` is a function that accepts the converted results, not a function that accepts a function that produces the converted results.